xsd/NeTEx_publication.xsd

```diff
@@ -7047,27 +7171,6 @@ Correct COnstraints for PointOnRoute
 			<xsd:field xpath="@id"/>
 			<xsd:field xpath="@version"/>
 		</xsd:key>
-		<!-- =====TypeOfFlexibleService============================== -->
```

Comment:
Intentional?

Reply:
Was duplicate. So I guess yes. But this is something Nick and Christophe should check..